docs/docs/forms.mdx

```diff
@@ -32,13 +32,42 @@ The following example form `restaurant_form` will fill the slot
 ```yaml-rasa
 forms:
   restaurant_form:
-    cuisine:
-      - type: from_entity
-        entity: cuisine
-    num_people:
-      - type: from_entity
-        entity: number
+    global_not_intent: intent
+    required_slots:
+        cuisine:
+          - type: from_entity
+            entity: cuisine
+        num_people:
+          - type: from_entity
+            entity: number
+```
+
+There is also an optional parameter that can be defined in each form which is called
+`global_not_intent` parameter. It can be added under the form_name and its value will
+affect all slots that are present in that form. More specifically, it will update/add
+the `not_intent` key of all individual slots with the specified value (or list of values)
+found in the global parameter.
+
+For example, if you want to fill the `not_intent` key of all slots with `intent` then
```

Comment:
I think this wording is a little confusing. I would prefer to focus on the fact that it means non of the slots will be filled with the intent `intent` (maybe rename too), rather than the fact it "fills" the key "nont_intent" for each slot.